content/300-guides/050-database/820-multi-schema.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: How to use Prisma with multiple database schemas
+metaDescription: How to use Prisma with multiple database schemas
+tocDepth: 2
+---
+
+<TopBlock>
+
+<Admonition type="warning">
+
+Multiple database schema support is currently a very early, experimental preview feature. Introspection and applying your Prisma schema to your database with `migrate dev` and `db push` are currently not supported. If you rely on them, please do not try this preview feature in your main branch.
+
+</Admonition>
+
+<Admonition type="warning">
+
+Multiple database schema support is currently only available with the PostgreSQL and SQL Server connectors.
```

Comment:
Is this correct?

Reply:
@tomhoule

Reply:
[Checked this on Slack](https://prisma-company.slack.com/archives/C02FNFLDUS3/p1662631064731419), it's correct